methods/sort-by.html

```diff
@@ -0,0 +1,65 @@
+<script>
+  init({
+    title: 'Sort By',
+    desc: 'Sort data in table: `$table.bootstrapTable(\'sortBy\', {field: 'name', sortOrder: 'asc'})`.',
+    links: ['bootstrap-table.min.css'],
+    scripts: ['bootstrap-table.min.js']
+  })
+</script>
+
+Sort by:
+<select id="sort">
+  <option value="name_asc">Name ascending</option>
+  <option value="name_desc">Name descending</option>
+  <option value="stargazers_asc">Stargazers low - high</option>
+  <option value="stargazers_desc">Stargazers high - low</option>
+  <option value="forks_asc">Forks low - high</option>
+  <option value="forks_desc">Forks high - low</option>
+</select>
+
+<table
+  id="table"
+  data-toggle="table"
+  data-sort-name="github.name"
+  data-url="json/data3.json">
+  <thead>
+    <tr>
+      <th data-field="github.name">Name</th>
+      <th data-field="github.count.stargazers">Stargazers</th>
+      <th data-field="github.count.forks">Forks</th>
+      <th data-field="github.description">Description</th>
+    </tr>
+  </thead>
+</table>
+
+<script>
+  $(function() {
+    $("#sort").change(function() {
+      var field = '';
+      var sortOrder = '';
```

Comment:
* update to use `mounted`
* remove `;` 
* add an empty line:
```js
  function mounted() {
    $("#sort").change(function() {
      var field = ''
      var sortOrder = ''

      if (this.value == 'name_asc') {

```

Reply:
Updated.